common/rpc/RpcChannel.cpp

```diff
@@ -77,8 +77,15 @@ class OutstandingResponse {
    * These are Requests on the client end that haven't completed yet.
    */
  public:
-    OutstandingResponse() {}
```

Comment:
That's what's causing part of the problem. There are no sane defaults to provide.